proxy/http2/Http2ConnectionState.h

```diff
@@ -38,7 +38,7 @@ enum Http2SendADataFrameResult {
   HTTP2_SEND_A_DATA_FRAME_DONE       = 3,
 };
 
-enum Http2ShutdownState { NOT_INITIATED, INITIATED, IN_PROGRESS };
+enum Http2ShutdownState { NOT_PLANNED, NOT_INITIATED, INITIATED, IN_PROGRESS };
```

Comment:
There's only one other `schedule_in` in HTTP/2 and that keeps hold of the returned `Action` so it can be canceled. I don't this this usage is safe.